### PR TITLE
Feature/show history of walks

### PR DESCRIPTION
**Pull request by yannhn**

### CODE REVIEW

File: src/components/HistoryModal/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import {ModalContainer, Modal, StyledHistoryModalHeader, StyledHistoryModalButton} from './styled';
+
+export default function HistoryModal({children, cancelHistoryForm}) {
+	return (
+		<ModalContainer>
+			<Modal open>
+				<StyledHistoryModalHeader>
+					<h2>History</h2>
+					<StyledHistoryModalButton type="button" onClick={cancelHistoryForm}>
```
Comment (macfrei) [inline src/components/HistoryModal/index.js:9]: When passing a callback function, usually the prefix `on` is used to distinguish them between all the other props.
```suggestion
export default function HistoryModal({children, onCancelHistoryForm}) {
	return (
		<ModalContainer>
			<Modal open>
				<StyledHistoryModalHeader>
					<h2>History</h2>
					<StyledHistoryModalButton type="button" onClick={onCancelHistoryForm}>
```

File: src/components/WalkCard/index.js [hunk 2]
```diff
@@ -57,6 +70,26 @@ export default function WalkCard() {
...
+						.map(date => (
+							<section key={date}>
+								<h2>{date}</h2>
+								{walkItem
+									.filter(walk => walk.date === date)
+									.map(walk => (
+										<section key={walk.id}>
+											<p>Duration: {walk.duration} h/m</p>
+											<p>When started: {walk.startTime}</p>
+											<hr></hr>
```
Comment (macfrei) [inline src/components/WalkCard/index.js:86]: I would recommend to not use extra HTML elements but CSS for styling :)

**Review by macfrei — COMMENTED** [review]
Good job, added a few comments :)

**Comment by yannhn** [comment]
Thanks for the feedback @macfrei ! I already implemented it. Thanks!

**Pull request merged**